src/cleanup_rules/swift/rules.toml

```diff
@@ -874,6 +874,90 @@ queries = [
     )""",
 ]
 
+#
+# Before 
+#   true == true
+# After 
+#   true
+#
+[[rules]]
+name = "boolean_equality_expression_both_true"
+query = """(
+    (equality_expression
+            lhs: (boolean_literal) @left
+            rhs: (boolean_literal) @right
+    ) @cond
+    (#eq? @left "true")
```

Comment:
```suggestion
    (#eq? @left @right)
```

Reply:
Oh this is nice, I had the misconception that rhs of eq has to be a hole variable. This will simplify it.